indico/modules/groups/core.py

```diff
@@ -162,7 +164,9 @@ def name(self):
     @cached_property
     def group(self):
         """The underlying :class:`.LocalGroup`"""
-        return LocalGroup.get(self.id)
+        if config.LOCAL_GROUPS:
```

Comment:
I think doing it the other way round would be clearer, i.e. if local groups are disabled, return none early.

BTW, I'd add a comment here mentioning why we are doing this:

```
# pretend local groups do not exist if they are disabled
# this way they'll be rejected in acl fields
```